tools/editor/scenes_dock.cpp

```diff
@@ -1416,6 +1428,10 @@ void ScenesDock::_files_list_rmb_select(int p_item,const Vector2& p_pos) {
 
 	file_options->add_item(TTR("Delete"),FILE_REMOVE);
 	//file_options->add_item(TTR("Info"),FILE_INFO);
+
+	file_options->add_separator();
+	file_options->add_item(TTR("Show In System"),FILE_SHOW_IN_EXPLORER);
```

Comment:
IMO the name may be a bit confusing. Something like _"Show In File Manager"_ could be better.